app/src/main/java/com/lagradost/cloudstream3/ui/settings/extensions/PluginAdapter.kt

```diff
@@ -135,6 +135,7 @@ class PluginAdapter(
         ) {
             val metadata = data.plugin.second
             val disabled = metadata.status == PROVIDER_STATUS_DOWN
+            val name = if (metadata.name.contains("Provider")) metadata.name.replace("Provider", "") else metadata.name
```

Comment:
The if statement isn't required and I think replaceSuffix is more appropriate, but good thinking :+1:

Reply:
Do i need to change it?